src/pages/home-page/HomePage.tsx

```diff
@@ -1,33 +1,67 @@
 import React, { useState } from "react";
 import { useOutletContext } from "react-router-dom";
-import { PokemonsList } from "./PokemonsList";
 import { SearchBar } from "@/design-system/generic-componenets/SearchBar";
+import { GenericDropDown} from "@/design-system/generic-componenets/GenericDropDown";
+import type { Options } from "@/design-system/generic-componenets/GenericDropDown";
+import { usePokemonsData } from "@/hooks/usePokemonsData";
+import type { SortOption } from "@/hooks/usePokemonsData";
+import { PokemonsList } from "./PokemonsList";
 
 export const HomePage: React.FC = () => {
   const { showMyPokemons } = useOutletContext<{ showMyPokemons: boolean }>();
-  const [searchTerm, setSearchTerm] = useState("");
+  const [searchTerm, setSearchTerm] = useState<string>("");
+  const [sortOption, setSortOption] = useState<SortOption>("alpha-asc");
+
   const title = showMyPokemons ? "My Pokemons" : "All Pokemons";
```

Comment:
```suggestion
const pokemonTableTitle = showMyPokemons
  ? PokemonTableTitles.mine
  : PokemonTableTitles.all;
```

create enum for that if u use that in more places except here

enum PokemonTableTitle {
  All = "All Pokemons",
  Mine = "My Pokemons",
}